### ISSUE TITLE
LaTeX manual not built, but make install tries to install it

### ISSUE THREAD

**Issue by eyalroz** [doxygen/doxygen#7810]
After CMake'ing and building on my Devuan GNU/Linux 3.0 Beowulf, I'm trying to `make install`. However - I'm getting the following error:
```
CMake Error at doc/cmake_install.cmake:58 (file):
  file INSTALL cannot find
  "/tmp/doxygen-1.8.18/build/latex/doxygen_manual.pdf".
Call Stack (most recent call first):
  cmake_install.cmake:50 (include)
```
and indeed - no `latex` subdirectory exists in my `build` directory.

Note: My CMake option settings:
```
 build_app                        ON                                                                                                                                                                                                          
 build_doc                        ON                                                                                                                                                                                                          
 build_doc_chm                    OFF                                                                                                                                                                                                         
 build_parse                      ON                                                                                                                                                                                                          
 build_search                     OFF                                                                                                                                                                                                         
 build_wizard                     OFF                                                                                                                                                                                                         
 build_xmlparser                  OFF                                                                                                                                                                                                         
 english_only                     OFF                                                                                                                                                                                                         
 force_qt4                        OFF                                                                                                                                                                                                         
 use_libclang                     ON                                                                                                                                                                                                          
 use_sqlite3                      OFF                                                                                                                                                                                                         
 win_static                       OFF        
```

**Comment by albert-github** [comment]
Did you try to build the documentation (`make docs`)?
Still this should not happen as in the manual it is mentioned as optional step (though in the manual I also see a strange `./configure` that should not be present)

**Comment by eyalroz** [comment]
> Did you try to build the documentation (make docs)?

No. But I also didn't make app, make parse etc. If I turned those options on, `make` should build everything I need for an installation.

**Comment by albert-github** [comment]
It is true that app, parse  etc. are build automatically together with the doxygen executable, the extra time involved is not that big. Building the documentation takes a bit longer and it is not always necessary to rebuild the documentation, ,to the best of my knowledge this is the reason to separate this though at least a sensible message should be given.

**Comment by eyalroz** [comment]
I'd say that `make install` should only install the documentation if it was built. So a message could say something like "Documentation not built - skipping its installation".

**Comment by albert-github** [comment]
I've just pushed a proposed patch, pull request #7821

**Comment by albert-github** [comment]
Code has been integrated in master on github (please don't close the issue as this will be done at the moment of an official release).

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.8.19.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
